### PR TITLE
Update README.md with more complete instructions

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,12 +93,13 @@ When editing any css or table of contents organization changes, the livereload e
...
 
 ### Versioning
+
 - Build for versioning (must be on master) branch: `make multiversion`
 - To view version metadata: `make multiversionpreview`
 
 ## Contributing to the docs
 
-If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an `issue <https://github.com/Start9Labs/documentation/issues>`__ .
+If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an `issue <https://github.com/Start9Labs/documentation/issues>`\_\_ .
```
Comment (gStart9) [inline README.md]:
```suggestion
If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an `issue <https://github.com/Start9Labs/documentation/issues>`.
```

File: README.md [hunk 2]
```diff
@@ -73,12 +93,13 @@ When editing any css or table of contents organization changes, the livereload e
...
 
 ### Versioning
+
 - Build for versioning (must be on master) branch: `make multiversion`
 - To view version metadata: `make multiversionpreview`
 
 ## Contributing to the docs
 
-If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an `issue <https://github.com/Start9Labs/documentation/issues>`__ .
+If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an `issue <https://github.com/Start9Labs/documentation/issues>`.
```
Comment (gStart9) [inline README.md]:
```suggestion
If you notice a mistake in the docs, or you have an idea for improving them, we would like to know about it. You can either fix the mistake yourself and submit a pull request, or you can report the mistake by submitting an [issue](https://github.com/Start9Labs/documentation/issues).
```

File: README.md [hunk 3]
```diff
@@ -50,13 +50,33 @@ git clone https://github.com/Start9Labs/documentation.git && cd documentation
...
+1. Prep NVM:
+
+```
+nvm use v16.11 || nvm install v16.11 && nvm use v16.11
+```
+
+1. Ensure poetry is updated
+
+```
+poetry update
```
Comment (kn0wmad) [inline README.md:62]: I'm guessing this is not accomplished by the poetry install in the setup script?

File: README.md [hunk 4]
```diff
@@ -50,13 +50,33 @@ git clone https://github.com/Start9Labs/documentation.git && cd documentation
...
 ```
 chmod a+x site/_utils/setup.sh
 ./site/_utils/setup.sh
 ```
 
+1. Make sure sphinx packages in the package.lock file are preset, for consistency:
+
+```
+cd sphinx-scylladb-theme
+npm ci
```
Comment (kn0wmad) [inline README.md:76]: `ci` or `i` ?

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Couple questions

File: README.md (hunk 4 above)
Comment (gStart9) [reply README.md:76]: Lucy says:
npm ci will just make sure that the packages in the package.lock file are present, which is best for consistency. package.lock would get updated if I (or someone) made changes to the submodule that required new packages/package versions

File: README.md (hunk 3 above)
Comment (gStart9) [reply README.md:62]: I don't think so, or at least it doesn't obviously, and certainly not explicitly (as can be seen here):

https://install.python-poetry.org/

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**